src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -2214,6 +2214,15 @@ register_xclbin(const xrt::xclbin&)
   throw xrt_core::ishim::not_supported_error{__func__};
 }
 
+//Exec Buf with ctx handle.
+void
+shim::
+exec_buf(xclBufferHandle boh, const xrt::hw_context& hwctx)
+{
+  // TODO: Implement new function, for now just call legacy xclExecBuf().
```

Comment:
So, there will be a follow up PR to really address the issue specified in the PR description?

Reply:
Yes Max. Saif is parallelly making that change.

Reply:
OK, got it. It'd be nice if we can make it clear in the PR description.

Reply:
updated PR description.